src/components/CookieWarning.js

```diff
@@ -13,17 +14,55 @@ import { authStore } from 'src/libs/state'
 export const cookiesAcceptedKey = 'cookiesAccepted'
 
 const CookieWarning = () => {
+  const animTime = 0.3
   const signal = useCancellation()
+  const [showWarning, setShowWarning] = useState(false)
+  const [opacity, setOpacity] = useState(0)
   const { cookiesAccepted } = useStore(authStore)
+  const timeout = useRef()
 
   const acceptCookies = acceptedCookies => {
     authStore.update(_.set('cookiesAccepted', acceptedCookies))
   }
 
+  useEffect(() => {
+    if (cookiesAccepted === undefined) {
+      timeout.current = setTimeout(showComponent, 3000)
+    } else {
+      cookiesAccepted ? hideComponent() : showComponent()
+    }
+  })
+
+  const showComponent = () => {
+    setShowWarning(true)
+    if (timeout.current) {
+      clearTimeout(timeout.current)
+    }
+    timeout.current = setTimeout(() => {
+      setOpacity(1)
+      timeout.current = undefined
+    }, 0)
+  }
+
+  const hideComponent = () => {
+    setOpacity(0)
+    if (timeout.current) {
+      clearTimeout(timeout.current)
+    }
+    timeout.current = setTimeout(() => {
+      setShowWarning(false)
+      timeout.current = undefined
+    }, animTime * 1000)
+  }
+
   const rejectCookies = async () => {
     const cookies = document.cookie.split(';')
     acceptCookies(false)
-    await Ajax(signal).Runtimes.invalidateCookie()
+    try {
+      await Ajax(signal).Runtimes.invalidateCookie()
+    } catch (err) {
+      // Do Nothing
```

Comment:
Im sorry, I dont know how to answer this question, this was already there before - I guess we could remove the await!

Reply:
To clarify, I was suggesting something like
```js
Ajax(signal).Runtimes.invalidateCookie().catch(() => {})
```
to avoid the `await` while still silencing the error. I'm also fine with leaving the `await` as is.

Reply:
Oh I like that a lot better, I'll make this change!